apps/site/pages/docs/go-live/overview.mdx

```diff
@@ -44,8 +44,8 @@ To ensure that you have set up all configurations for your store to run smoothly
 <details>
 <summary>Ensure that your account edition is set to `vtex.edition-store@5.x` Edition app in your master workspace.</summary>
 
-| Management              | Reference                                                                                                                                                                       |
-| :---------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Management              | Reference                                                                                                                                                                    |
+| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Agency and VTEX Support |  If your account uses a different Edition App version, [open a support ticket](https://help.vtex.com/en/support) requesting the installation of the vtex.edition-store@5.x.  |
```

Comment:
> If your account uses a different Edition App version, [open a support ticket](https://help.vtex.com/en/support) requesting the installation of the vtex.edition-store@5.x.
Questions:
1. Does the recommendation also apply to Legacy stores? 
2. Should they have the editor set in `master`?